scripts/submit_eval_jobs.py

```diff
@@ -1,25 +1,35 @@
 import copy
 import subprocess
 import yaml
-import random
 import re
 import itertools
 from datetime import date
+import argparse
 
 today = date.today().strftime("%m%d%Y")
 
+parser = argparse.ArgumentParser()
+parser.add_argument("--workspace", type=str, default="hamishivi")
+parser.add_argument("--model_name", type=str, default="hf-opt-7B")
+parser.add_argument("--location", type=str, default=None)
+parser.add_argument("--beaker_subfolder", type=str, default=None)
+parser.add_argument("--cluster", type=str, default="ai2/allennlp-cirrascale")
+parser.add_argument("--num_gpus", type=int, default=1)
+parser.add_argument("--is_tuned", action="store_true")
+args = parser.parse_args()
+
+
+workspace = args.workspace
+model_type = "vanilla_lm" if not args.is_tuned else "tuned_lm"
+
 with open("beaker_configs/default_eval.yaml", 'r') as f:
     default_yaml = f.read()
 d1 = yaml.load(default_yaml, Loader=yaml.FullLoader)
 
-# cluster = "ai2/general-cirrascale"
-cluster = "ai2/allennlp-cirrascale"
-# cluster = "ai2/general-cirrascale-a100-80g-ib"
-# cluster = "ai2/prior-elanding"
-num_gpus = 1
+cluster = args.cluster
+num_gpus = args.num_gpus
```

Comment:
Later, the num_gpus will be dynamically changed based on the model size. So, the argument here might be confusing. Shall we move the hard-coded GPUs into some post init functions for these arguments?

Reply:
Yeah, I didn't realise this... I think maybe a logger that warns if you differ from suggestions?